documents/toolbox-population.md

```diff
@@ -0,0 +1,80 @@
+# Toolbox population
+
+## Toolbox items from unreferenced assemblies (all supported versions of Visual Studio)
+
+This approach requires registering with the Toolbox Controls Installer (TCI) in the Windows registry. [The TCI spec](https://www.microsoft.com/en-us/download/details.aspx?id=35536) explains how to register your assemblies. Once registered, your controls will be shown in the Toolbox for any compatible project, even projects that do not reference your assembly.
+
+### Limitations
+
+* This approach only works reliably for WPF .NET Framework assemblies. The controls in registered assemblies are enumerated via .NET Reflection which does not work reliably on .NET Core assemblies within a .NET Framework process such as Visual Studio.
+* Since registration requires modifying the registry, you need to provide an installer of some sort.
+
+## Toolbox items from referenced NuGet packages (Visual Studio 15.0 and later)
+
+If a NuGet package referenced by a XAML project contains a [tools\VisualStudioToolsManifest.xml file](https://docs.microsoft.com/en-us/nuget/guides/create-ui-controls), Toolbox will show the Toolbox items listed in the manifest.
+
+You can try this by adding a reference to your own package or to [our WPF .NET Core sample package](#how-to-build-our-wpf-net-core-sample-package).
+
+### Target Framework-specific Manifests
+
+In Visual Studio 16.6 and later, Toolbox population supports multiple VisualStudioToolsManifest.xml files per package -- the maniest in the tools root plus additional manifests for specific Target Framework Monikers (TFMs) in subdirectories of tools. The Toolbox will show the items from the manifest that best matches the target framework of the current project, falling back to the manifest in the tools root if there is no better match. The manifest in the tools root is also required for compatibility with older versions of Visual Studio.
+
+Here's an example:
+
+| File path                                         | Will be used for projects targetting...                     |
+| :------------------------------------------------ | :---------------------------------------------------------- |
+| \tools\net47\VisualStudioToolsManifest.xml        | .NET Framework >= 4.7                                       |
+| \tools\netcoreapp31\VisualStudioToolsManifest.xml | .NET Core >= 3.1                                            |
+| \tools\VisualStudioToolsManifest.xml              | .NET Framework < 4.7, .NET Core < 3.1, and other frameworks |
+
+## Toolbox items from unreferenced NuGet packages (Visual Studio 16.7 Preview 2 and later)
+
+If a NuGet package in a NuGet fallback folder contains a [tools\VisualStudioToolsManifest.xml file](https://docs.microsoft.com/en-us/nuget/guides/create-ui-controls), Toolbox will show the Toolbox items listed in the manifest for any project that is compatible with the package, even projects that do not reference that package.
```

Comment:
in a Nuget fallback folder (see [Getting Started](#getting-started) below) contains ...

Reply:
Added text and link